pkg/vm/vm_ops_maths_test.go

```diff
@@ -394,3 +394,71 @@ func TestShrOp(t *testing.T) {
 
 	assert.Equal(t, int64(2), item.Value().Int64())
 }
+
+func TestLtOp(t *testing.T) {
+
+	v := VM{}
+
+	a, err := stack.NewInt(big.NewInt(10))
+	if err != nil {
```

Comment:
It could be better to use https://godoc.org/github.com/stretchr/testify/assert#NoError
so that all code looks uniform.

Reply:
:+1: